### PR TITLE
V3 to V4 Migration Tool

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/developerportal/deploy/migrating-to-v4.md [hunk 1]
```diff
@@ -55,13 +46,188 @@ There are a few other differences between the way you develop and deploy apps in
 * Mendix Cloud v4 only supports TLS 1.2 or above for incoming requests
     * If you have external clients connecting _to_ your application running in the Mendix Cloud, these clients have to support TLS 1.2 or above to be able to make a successful connection
 
-## 4 Migrating the App
+## 3 Prerequisites
+
+There are two methods of doing the migration. For both of them you first need to ensure that your app is on at least **Mendix version 6.0**, otherwise the migration will fail – it is preferable to be on a supported version of Mendix: **Mendix version 7.0 and above**
```
Comment (jeohist) [inline content/developerportal/deploy/migrating-to-v4.md:51]: Should we recommend people to use the LTS version of Mendix 7, 7.23.x?

Comment (MarkvanMents) [reply content/developerportal/deploy/migrating-to-v4.md]: Have added this recommendation. Thanks.

File: content/developerportal/deploy/migrating-to-v4.md [hunk 2]
```diff
@@ -55,13 +46,192 @@ There are a few other differences between the way you develop and deploy apps in
...
+4. Select the source and target environments. In most cases, these will have the same name. For example, you want to replicate data from the V3 `production` environment to the V4 `production` environment.
+
+    ![Select V3 node and environment](attachments/migrating-to-v4/select-v3-node-to-migrate.png)
+
+5. Ensure that the app in the target V4 environment is stopped. You cannot replicate data into an app that is running.
+
+6. Click **Replicate data and files**.
+
+    The replication process will copy all the data in the database, and files based on `FileDocument` entities, such as images, which are stored in the S3 storage.
+
```
Comment (MarkvanMents) [inline content/developerportal/deploy/migrating-to-v4.md:120]: Add a warning here that the validation might impact the V3 database.
Is the impact only on reporting, or might it affect performance of the app?
See information in paper document describing the process.

File: content/developerportal/deploy/migrating-to-v4.md [hunk 3]
```diff
@@ -55,13 +46,197 @@ There are a few other differences between the way you develop and deploy apps in
...
     * If you have external clients connecting _to_ your application running in the Mendix Cloud, these clients have to support TLS 1.2 or above to be able to make a successful connection
 
-## 4 Migrating the App
+## 3 Prerequisites
+
+There are two methods of doing the migration. For both of them you first need to ensure that your app is on at least **Mendix version 6.0**, otherwise the migration will fail – it is preferable to be on a supported version of Mendix, **Mendix version 7.0 and above**. For the best level of support, we recommend that you use version **7.23** which is the [long term support](/releasenotes/studio-pro/lts-mts) version of Mendix version 7.
+
+For further prerequisites see the sections below.
+
+## 4 Using the Migration Tool
```
[the remark this answers is not in the document]
Comment (MarkvanMents) [reply content/developerportal/deploy/migrating-to-v4.md:55]: Added

**Comment by MarkvanMents** [comment]
@NielsTrots 
Hi Niels,
This is the documentation for the v3 to v4 migration tool. Do you want to review it from a support perspective, in case there are questions coming to you?

File: content/developerportal/deploy/migrating-to-v4.md [hunk 4]
```diff
@@ -55,13 +46,201 @@ There are a few other differences between the way you develop and deploy apps in
...
+{{% image_container width="50%" %}}
+![Overview of Migration Process](attachments/migrating-to-v4/migration-process-overview.png)
+{{% /image_container %}}
+
+### 4.1 Prerequisites
+
+To migrate your app from Mendix Cloud V3 to Mendix Cloud V4, you need to following prerequisites:
+
+* Have a [Mendix Cloud](mendix-cloud-deploy) v4 node available. This **must have the name `<current-app-name>-v4`**. To request a licensed v4 Cloud Node, contact your Customer Success Manager (CSM). In most cases, the node will have the same environments as the V3 app you are migrating
+* Have the [Technical Contact](/developerportal/collaborate/app-roles#technical-contact) role for both your existing v3 and available v4 Cloud Nodes
```
[the remark this answers is not in the document]
Comment (MarkvanMents) [reply content/developerportal/deploy/migrating-to-v4.md:76]: It says that you should be Technical Contact on both the apps, so I'm not sure of the value of this change.

File: content/developerportal/deploy/migrating-to-v4.md (hunk 2 above)
Comment (MarkvanMents) [reply content/developerportal/deploy/migrating-to-v4.md:124]: Discussed and decided not to mention unless customers report issues.

**Comment by MarkvanMents** [comment]
Approved over Slack.

**Pull request merged**